migrator/migrator.go

```diff
@@ -169,14 +174,21 @@ func (d *D2pMigrator) PrepareImages(ctx context.Context) error {
 		_, err = ctrdCli.GetImage(ctx, imageName)
 		if err == nil {
 			logrus.Infof("image %s has been downloaded, skip pull image", imageName)
-		} else {
-			logrus.Infof("Start pull image: %s", imageName)
-			if err := ctrdCli.PullImage(ctx, imageName); err != nil {
-				logrus.Errorf("failed to pull image %s: %v", imageName, err)
-				continue
-			}
-			logrus.Infof("End pull image: %s", imageName)
+			continue
```

Comment:
all images will not download?

Reply:
This statement means "this image already download under pouchd"